api/src/Entity/ContentNode/MultiSelectOption.php

```diff
@@ -2,35 +2,68 @@
 
 namespace App\Entity\ContentNode;
 
+use ApiPlatform\Core\Annotation\ApiFilter;
 use ApiPlatform\Core\Annotation\ApiProperty;
 use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\SearchFilter;
 use App\Entity\BaseEntity;
+use App\Entity\BelongsToCampInterface;
+use App\Entity\Camp;
 use App\Entity\SortableEntityInterface;
 use App\Entity\SortableEntityTrait;
+use App\Repository\MultiSelectOptionRepository;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 /**
- * @ORM\Entity
+ * @ORM\Entity(repositoryClass=MultiSelectOptionRepository::class)
  * @ORM\Table(name="content_node_multiselect_option")
- * @ApiResource(routePrefix="/content_node")]
  */
-class MultiSelectOption extends BaseEntity implements SortableEntityInterface {
+#[ApiResource(
+    routePrefix: '/content_node',
+    collectionOperations: [
+        'get' => [
+            'security' => 'is_fully_authenticated()',
+        ],
+        // 'post' => creating new options is not allowed
```

Comment:
Mögliche Alternative: 
```
        'post' => [
            'security' => 'false',
        ],
```

Reply:
Ich habe die operation bei den Invitations einfach weggelassen, und einen test geschrieben (zwar nur für delete, aber immerhin), dass die operation nicht geht.
Ich habe da keine Präferenz, nur ein vorschlag von mir. (ok, vielleicht leicht zum test hin, aber die ist nur persönlich)

Reply:
Wenn wir die Methode ganz weglassen wird es in der OpenAPI-Dokumentation besser dokumentiert. Wenn du `security => false` schreibst hat man gemäss Swagger unter irgendwelchen Bedingungen die Option, diese Operation zu nutzen.
Daher wäre ich auch dafür, nur den Kommentar oder sogar gar nichts hier stehen zu lassen, und einen einzigen Test zu schreiben der auf 405 prüft. Habe ich bei ContentType und weiteren Entities auch schon so gemacht.